product_docs/docs/pgd/5/manual/index.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Deploying manually
+navigation:
+- 01-provisioning-hosts
+- 02-install-postgres
+- 03-configuring-repositories
+- 04-installing-software
+- 05-creating-cluster
+- 06-check-cluster
+- 07-configure-proxies
+- 08-using-pgd-cli
+---
+
+EDB offers automated PGD deployment using TPA (Trusted Postgres Architect) because generally they are more reliable than manual processes. 
+Consult [Deploying with TPA](../tpa/) for how to install TPA and use it's automated best-practice driven PGD deployment options for full details.
+Or refer to any of the [Quickstart walkthroughs](../quickstart/) which use TPA to get you up and running quickly.
+
+To complement automated installation, and to enable alternative installation and deployment processes, this section of the documentation looks at the basic operations needed to manually configure a three-node PGD cluster, with a local sub-group, PGD Proxy and PGD CLI.
```

Comment:
I've now gone halfway through a manual install and started installing packages.  At this point I realized that I very much have no idea if the systems I have provisioned for the install and the users I planned to perform the install with meet the minimum hardware specs and user privilege requirements for the various software that I will be installing and the users I will be using to perform the install.  This led me back to the beginning of the docs section, searching for a "requirements" section showing minimum h/w, priveleges, etc requirements ... which from what I can tell does not exist.  EPAS has a section like that (https://www.enterprisedb.com/docs/epas/latest/planning/planning_prerequisites/epas_requirements/#user-privileges). I'm not sure PGD has one but it needs one, esp. now that I am manually installing

Reply:
We don't per se have a section like that but we do outline user requirements on page one of the manual install under "Create an admin user"

Reply:
Got it...could the admin user piece be moved to a "requirements" section?  Why is passwordless SSH required?